src/javascripts/components/flightDashboard/singleFlight/singleFlight.js

```diff
@@ -16,13 +17,22 @@ const viewSingleFlight = (e) => {
   <ul class="list-group list-group-flush">
     <li class="list-group-item">Origin: ${flight.origin.data.name}</li>
     <li class="list-group-item">Destination: ${flight.destination.data.name}</li>
```

Comment:
If this were me I’d refactor this so you don’t need to do `thing.data.property` (So change the smash to make flight.plane = response.data or whatever)